packages/functional-tests/src/app.ts

```diff
@@ -58,14 +60,18 @@ export default class App {
         } else {
             testName = this.params.test as string;
         }
-
-        await this.startTest(testName);
+        if (testName) {
+            await this.startTest(testName);
+            this.rpc.send('functional-test:close-connection');
+        } else {
+            await this.launchTestBrowser();
+        }
     }
-
     private userLeft = (user: MRESDK.User) => {
         console.log(`user-left: ${user.name}, ${user.id}`);
     }
 
+

```

Comment:
This extra newline should be flagged by tslint.

Reply:
fixed - lint wasn't enabled by default in this package, which is why I missed it